src/main/java/com/gd/clinic/entities/Doctor.java

```diff
@@ -0,0 +1,29 @@
+package com.gd.clinic.entities;
+
+import com.sun.xml.bind.v2.TODO;
+import lombok.*;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "Doctor")
+public class Doctor {
```

Comment:
we have `users` table for this. so admin, doctor and nurse is basically the same entity, just with different roles

Reply:
Ok, the entity and repo have been removed this commit. Since the `Users` entity  is in the `security` branch, this will have to be implemented once `security` is merged into `main`